src/compiler/binder.ts

```diff
@@ -2537,12 +2541,11 @@ namespace ts {
                 (namespaceSymbol.members || (namespaceSymbol.members = createSymbolTable())) :
                 (namespaceSymbol.exports || (namespaceSymbol.exports = createSymbolTable()));
 
-            // Declare the method/property
-            const jsContainerFlag = isToplevelNamespaceableInitializer ? SymbolFlags.JSContainer : 0;

```

Comment:
this is just a refactoring that moves jsContainerFlag's usage into the call to declareSymbol

Reply:
Well, now it just puts JSContainer on everything regardless.